src/controllers/AuthController.js

```diff
@@ -37,9 +39,7 @@ class AuthController {
         .exec();
 
     const username = displayName.replace(/\s+/g, '') + id.substr(0, 5);
-    const token = jwt.sign({ email, username }, process.env.SECRET, {
-      expiresIn: '2d',
-    });
+    const token = encrypt.generateToken({ email, username });
```

Comment:
@Karlmusingo Why did you change `jwt` to `encrypt`?

Reply:
I am using using at different places, I just put it in the helpers folder.

Reply:
OK, but you should only sign using the id without providing the username and email